addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1125,6 +1125,10 @@ const Wysiwyg = Widget.extend({
             core.bus.trigger('activate_image_link_tool');
             return;
         }
+        // Avoid invoking toggleLinkTools for videos.
+        if (targetEl.parentNode.closest(".media_iframe_video")) {
+            return;
```

Comment:
It looks like it works, but I'm afraid there is more to it than that.
If you:
- insert a video,
- click on it,
- Ctrl + A,
- Ctrl + K
=> it breaks the layout (the iframe's height is resized) and the DOM (a link tag is added and set around the whole video block structure, and the iframe is duplicated, once within the link tag, once outside of it).
You can then just click on the buggy iframe to reset it to fullsize, but you end up with 2 iframes in the DOM (you only see one because they're both positioned absolutely).